### ISSUE TITLE
Display  edited badge on show page

### ISSUE THREAD

**Issue by shanitajohn** [curationexperts/epigaea#414]
**STORY**
As an administrator, once I have edited a work and saved my draft the "Edited" badge should also display on the item show page along with "Public" so that others can see the status. 

**ACCEPTANCE CRITERIA**
- [x] Show edited badge on single item view page
- [x] Capitalize "E" in "edited"

**MOCKUP**
When this is true... 
![edited_dashboard.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/56a796b4884b1f13002fff7c/125516c66e82c728ace21e0d46ed9093378787968be8ac4ba652f1862d046b313916ac217586f76ae5002e585e0a4efd455e5c11fcfdd87ea6b43d78890e0ee8d3394d68d47f40f2ecdd42e64bef62994597b7b4804dc5e83a2de58b88483e1a0c1a9bb2.png)

This is true...
![edited_badge_singleitem.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/56a796b4884b1f13002fff7c/125516c66e82c728ace21e0d46ed9093378787968de8bb44a162e39d27412b762010a62632c9a138ad4d71040c1b5ca6535d5645a9f9d82fa0b569708e0d5eedd36c5777943814aeec9003ad43eb7edf10d8eff6df0ed2e22f24e1818e4a3e190d1a8aff5532a99848da15.png)

**Comment by little9** [comment]
@mark-dce @no-reply @bess just so I'm clear about this, 'edited' status means that `work.has_draft?` is true and 'published' is `work.has_draft?` is false? That's what those statues mean on the edit page, should I be checking the state of the work in some other way on the show page? 

Do we only have two states? Right now there are only two states on the draft controls.

**Comment by mark-dce** [comment]
There should be three states:
[blue] unpublished: workflow state = unpublished
[green] published: workflow state = published && has draft = false
[yellow] edited: workflow state = published && has draft = true

**Comment by no-reply** [comment]
@little9: I think this was meant to have been closed by 97c39457bb, but @shanitajohn and I have found a bug. To reproduce

  - Create an item.
  - Make draft edits.
  - Click the breadcrumb to the item show view.
  - Click `Edit`
  - Revert the draft
  - Return to the item show view
  - The badge reads "Edited".

The indexed `has_draft` value remains true in this case.

Also, I've noticed that `Has Draft` appears at the bottom of the form. Could we remove this field?
